cmd/server-main.go

```diff
@@ -80,7 +80,7 @@ EXAMPLES:
 // Check for updates and print a notification message
 func checkUpdate(mode string) {
 	// Its OK to ignore any errors during getUpdateInfo() here.
-	if older, downloadURL, err := getUpdateInfo(1*time.Second, mode); err == nil {
+	if older, downloadURL, err := getUpdateInfo(mode, 1*time.Second); err == nil {
```

Comment:
Now I don't see any improvement/fixes the PR does than another way to implement with a struct.